docs/src/theme.js

```diff
@@ -0,0 +1,134 @@
+//  Copyright (c) 2018-present, GM Cruise LLC
+//
+//  This source code is licensed under the Apache License, Version 2.0,
+//  found in the LICENSE file in the root directory of this source tree.
+//  You may not use this file except in compliance with the License.
+
+// basic palette colors
+const baseColors = [
+  { label: "dark", value: "#1f1e27" },
+  { label: "white", value: "#ffffff" },
+  { label: "gray", value: "#88878a" },
+  { label: "taupe", value: "#f7f7f3" },
+  { label: "red", value: "#f24366" },
+  { label: "red1", value: "#ff7c96" },
+  { label: "red2", value: "#ffaabb" },
+  { label: "blue", value: "#1b83ec" },
+  { label: "blue1", value: "#4e98e2" },
+  { label: "blue2", value: "#77aadd" },
+  { label: "paleGreen", value: "#b4cc00" },
+  { label: "paleGreen1", value: "#cad660" },
+  { label: "paleGreen2", value: "#cdd67e" },
+  { label: "orange", value: "#ea531f" },
+  { label: "orange1", value: "#f5774d" },
+  { label: "orange2", value: "#faa487" },
+  { label: "cyan", value: "#22b5ff" },
+  { label: "cyan1", value: "#61cbff" },
+  { label: "cyan2", value: "#99ddff" },
+  { label: "green", value: "#05d27d" },
+  { label: "green1", value: "#5cd6a9" },
+  { label: "green2", value: "#8de0c9" },
+  { label: "purple", value: "#6e51ee" },
+  { label: "purple1", value: "#a395e2" },
+  { label: "purple2", value: "#c7c0e7" },
+  { label: "yellow", value: "#edcc28" },
+  { label: "yellow1", value: "#f7df71" },
+  { label: "yellow2", value: "#f1e4a9" },
+];
+
+const opacities = [
+  { value: 90, hex: "E6" },
+  { value: 80, hex: "CC" },
+  { value: 70, hex: "B3" },
+  { value: 60, hex: "99" },
+  { value: 50, hex: "80" },
+  { value: 40, hex: "66" },
+  { value: 30, hex: "4D" },
+  { value: 20, hex: "33" },
+  { value: 10, hex: "1A" },
+  { value: "05", hex: "0D" },
```

Comment:
Why is value a string here?

Reply:
Ah I see the function below now. Maybe just make them all strings for consistency then? A bit confusing otherwise.

Reply:
Good catch! Actually `color.white5' should be valid and straightforward,  we can just remove the string and keep all numbers.